### PR TITLE
Add banners to repo READMEs

### PR DESCRIPTION
**Pull request by nonparibus**
## 🌟 What is the purpose of this PR?

Add various cover images to the README, and shields in a couple of places they were missing.

## How to test

Visit and check the image/shields on the following pages:
1. https://github.com/hashintel/hash/blob/dw/misc-changes/packages/README.md
1. https://github.com/hashintel/hash/blob/dw/misc-changes/apps/README.md
1. https://github.com/hashintel/hash/blob/dw/misc-changes/blocks/README.md
1. https://github.com/hashintel/hash/blob/dw/misc-changes/infra/README.md
1. https://github.com/hashintel/hash/blob/dw/misc-changes/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
-# HASH
+[github_banner]: https://hash.dev/?utm_medium=organic&utm_source=github_readme_hash-repo_root
+[github_star]: https://github.com/hashintel/hash#
```
Comment (kachkaev) [inline README.md:2]:
```suggestion
[github_star]: https://github.com/hashintel/hash
```

? (here and elsewhere)

**Review by kachkaev — COMMENTED** [review]
The banners look nice! I noticed them being a bit slow to load on my mid-quality internet. Is there any way we can compress images a bit?

Screenshot 2022-12-20 at 16 08 38
Screenshot 2022-12-20 at 16 07 49

Also I'm not sure about the value of `[label]: url`, but I don’t mind using them.

File: README.md (hunk 1 above)
Comment (nonparibus) [reply README.md:2]: I don't want to reload the pages in all instance, but to jump the user up to the top of the page where the star button is (if they're on that page already). This achieves that.

**Review by kachkaev — DISMISSED** [review]

**Comment by nonparibus** [comment]
Switching to Cloudflare Images doesn't actually seem to have solved the file serving issue ^ It's still serving up the full-size ones to GitHub which they are then cacheing. I'm going to manually compress and re-upload.

**Review by kachkaev — APPROVED** [review]
Nice banners, great to see them compressed 👍

**Pull request merged**